public/pages/DetectorsList/List/List.tsx

```diff
@@ -143,20 +200,25 @@ export const DetectorList = (props: ListProps) => {
   };
 
   const isFilterApplied = !isEmpty(state.queryParams.search);
+  
   const pagination = {
     pageIndex: state.page,
     pageSize: state.queryParams.size,
     totalItemCount: Math.min(MAX_DETECTORS, totalDetectors),
     pageSizeOptions: [5, 10, 20, 50],
   };
+  
   return (
     <EuiPage>
+      <SideBar />
```

Comment:
Minor. no need to add SideBar here.

Reply:
What is the reason we don't need this here? From my understanding, this page and the dashboard have the sidebar for navigation.

Reply:
Detector list page will be linked to the Main page and the Main page already has sidebar , so no need to add sidebar on other pages.

Reply:
Got it, I removed.